pkg/ui/src/views/reports/containers/debug/index.tsx

```diff
@@ -0,0 +1,202 @@
+import _ from "lodash";
+import React from "react";
+
+function DebugTableLink(props: { name: string, url: string, note?: string }) {
+  return (
+    <tr className="debug-inner-table__row">
+      <td className="debug-inner-table__cell">
+        <a className="debug-link" href={props.url}>{props.name}</a>
+      </td>
+      <td className="debug-inner-table__cell--notes">
+        {_.isNil(props.note) ? props.url : props.note}
+      </td>
+    </tr>
+  );
+}
+
+function DebugTableRow(props: { title: string, children?: React.ReactNode }) {
+  return (
+    <tr className="debug-table__row">
+      <th className="debug-table__cell debug-table__cell--header">{props.title}</th>
+      <td className="debug-table__cell">
+        <table className="debug-inner-table">
+          <tbody>
+            {props.children}
+          </tbody>
+        </table>
+      </td>
+    </tr>
+  );
+}
+
+function DebugTable(props: { heading: string, children?: React.ReactNode }) {
+  return (
+    <div>
+      <h3>{props.heading}</h3>
+      <table className="debug-table">
+        <tbody>
+          {props.children}
+        </tbody>
+      </table>
+    </div>
+  );
+}
+
+export default function Debug() {
+  return (
+    <div className="section">
+      <h2>Advanced Debugging</h2>
+      <DebugTable heading="Reports">
+        <DebugTableRow title="Node Diagnostics">
+          <DebugTableLink name="All Nodes" url="#/reports/nodes" />
+          <DebugTableLink
+            name="Nodes filtered by node IDs"
+            url="#/reports/nodes?node_ids=1,2"
+            note="#/reports/nodes?node_ids=[node_id{,node_id...}]"
+          />
+          <DebugTableLink
+            name="Nodes filtered by locality (regex)"
+            url="#/reports/nodes?locality=region=us-east"
+            note="#/reports/nodes?locality=[regex]"
+          />
+        </DebugTableRow>
+        <DebugTableRow title="Network">
+          <DebugTableLink name="Latency (on all nodes)" url="#/reports/network" />
+          <DebugTableLink
+            name="Latency filtered by node IDs"
+            url="#/reports/network?node_ids=1,2"
+            note="#/reports/network?node_ids=[node_id{,node_id...}]"
+          />
+          <DebugTableLink
+            name="Latency filtered by locality (regex)"
+            url="#/reports/network?locality=region=us-east"
+            note="#/reports/network?locality=[regex]"
+          />
+        </DebugTableRow>
+        <DebugTableRow title="Security">
+          <DebugTableLink name="Certificates on this node" url="#/reports/certificates/local" />
+          <DebugTableLink
+            name="Certificates on a specific node"
+            url="#/reports/network?node_ids=1,2"
+            note="#/reports/network?node_ids=[node_id{,node_id...}]"
+          />
+          <DebugTableLink
```

Comment:
ditto here: name is `latency filtered by locality`, but link is `certificates/1`